cmd/kubeadm/app/cmd/phases/certs.go

```diff
@@ -105,11 +106,28 @@ func newCertSubPhase(certSpec *certsphase.KubeadmCert, run func(c workflow.RunDa
 			certSpec.BaseName,
 			getSANDescription(certSpec),
 		),
-		Run: run,
+		Run:      run,
+		CmdFlags: getCertPhaseFlags(certSpec.Name),
 	}
 	return phase
 }
 
+func getCertPhaseFlags(name string) []string {
+	flags := []string{
+		"config",
```

Comment:
This should all be const-i-fied in some central place with a comment block outlining the order.